src/libraries/System.Private.Xml/src/System/Xml/Core/XmlTextEncoder.cs

```diff
@@ -15,6 +15,7 @@ namespace System.Xml
     // it will replace special characters with entities whenever necessary.
     internal sealed class XmlTextEncoder
     {
+        private const int ChunkSize = 256;
```

Comment:
This is only used by `WriteStringFragment` so I'd just define it as a local there.